infrastructure/main.tf

```diff
@@ -39,7 +39,7 @@ module "citizen-frontend" {
     WEBSITE_NODE_DEFAULT_VERSION = "8.9.0"
 
     // Node specific vars
-    NODE_ENV = "${var.env}"
+    NODE_ENV = "${contains(list("dev", "demo", "test", "prod"), var.env) ? var.env : "dev"}"
```

Comment:
I would probably just go for a `== prod` here not sure if we get any benefit from the others, considering they probably won't ever be set to those values, at least for test

Reply:
Makes sense 👍